architecture/topics/router_environment_variables.adoc

```diff
@@ -61,6 +61,7 @@ and "-". The default is the hashed internal key name for the route.
 |`ROUTER_SYSLOG_ADDRESS` |  | Address to send log messages. Disabled if empty.
 |`ROUTER_SYSLOG_FORMAT` | | If set, override the default log format used by underlying router implementation. Its value should conform with underlying router implementation's specification.
 |`ROUTER_TCP_BALANCE_SCHEME` | source | xref:load-balancing[Load-balancing strategy] for multiple endpoints for pass-through routes. Available options are `source`, `roundrobin`, or `leastconn`.
+|`ROUTER_THREADS` | | Specifies the number of threads for the HAProxy router.
```

Comment:
If not set, then should be one thread by default right ?
if set, does it has ranges ?

Reply:
If not set the threading option is disabled. 
It checks that the argument is bigger than 0

Reply:
Thank you @imcsk8.
I also leave comment in the card: https://trello.com/c/0thyZwBw/
please help check the link of test cases there and help review. Thanks.